src/planner.c

```diff
@@ -542,6 +561,8 @@ _planner_init(void)
 	get_relation_info_hook = timescaledb_get_relation_info_hook;
 	prev_create_upper_paths_hook = create_upper_paths_hook;
 	create_upper_paths_hook = timescale_create_upper_paths_hook;
+
+	ts_constraint_aware_append_init();
```

Comment:
I think this should go into init.c

Reply:
you sure? functions in init.c have a different naming scheme

Reply:
done